seeder/seeder.js

```diff
@@ -1,6 +1,7 @@
 const path = require('path');
 
-const dlcData = require('../data/1-dlcs/dlc.js');
+const gameNames = require('./data/2-titles/title');
```

Comment:
Where do these get seeded into the database? Have they been seeded yet?

Reply:
Ya so by requiring them in this file, the `init()` for both get run and the array of seeding data gets generated and exported. Mongo-seeding then imports everything in the `data` folder and does handles all the seeding